src/components/HelperErrorText/HelperErrorText.stories.mdx

```diff
@@ -30,7 +31,7 @@ import DSProvider from "../../theme/provider";
       table: { defaultValue: { summary: true } },
     },
     ariaLive: {
-      table: { defaultValue: { summary: "polite" } },
+      table: { defaultValue: { summary: "undefined" } },
```

Comment:
Should `undefined` be a string or just the type itself?

Reply:
I removed it altogether.